tests/functional/examples/test_notebooks_python.py

```diff
@@ -261,6 +261,7 @@ def test_geoimc_functional(notebooks, output_notebook, kernel_name, expected_val
 
 @pytest.mark.notebooks
 @pytest.mark.experimental
+@pytest.mark.skip(reason="xLearn pip package has installation incompatibilities")
```

Comment:
is this temporal? or do we want to skip only at some condition, like using `skipif("some_module" not in sys.modules, reason="'some_module' was not installed correctly maybe due to version conflict")`?

Reply:
In reality, it is a matter than someone has time to integrate xlearn in the core libraries